### PR TITLE
Avatar and badge component

### PR DESCRIPTION
**Pull request by shristabaruah**
--Added avatar component
--Added badge component
avatar1
avatar2
badge1
badge2

### CODE REVIEW

File: components/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                    <a href="">Modal</a>
+                    <a href="">Navigation</a>
+                    <a href="">Rating</a>
+                    <a href="">Slider</a>
+                    <a href="">Toast</a>
+                    
+                </div>
+            </div>
+        </div>
+        <div class="section">
```
Comment (ruchiragoswami) [inline components/avatar/avatar.html:60]: I suggest not to use keywords for class names. Example, 'section' is a tag in HTML and using the same word for the class name might create confusion.

**Review by ruchiragoswami — APPROVED** [review]
A general comment, wherever applicable, use 'section tag' so its not just 'div' everywhere. Overall good work Shrista :)

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                    <a href="">Card</a>
+                    <a href="">Image</a>
+                    <a href="">Input</a>
+                    <a href="">Lists</a>
+                    <a href="">Modal</a>
+                    <a href="">Navigation</a>
+                    <a href="">Rating</a>
+                    <a href="">Slider</a>
+                    <a href="">Toast</a>
+
```
Comment (BISINIGISHIVANI) [inline components/avatar/avatar.html:56]: improve  Identation

**Review by BISINIGISHIVANI — APPROVED** [review]
avoid spacing/indentation apart of it all good

**Pull request merged**